internal/cmd/logout.go

```diff
@@ -42,11 +44,17 @@ $ infra logout --all --clear`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 1 {
 				if all {
-					fmt.Fprintf(os.Stderr, "  Server is already specified. Ignoring flag [--all] and logging out of server %s.\n", args[0])
-					all = false
+					return fmt.Errorf("Argument [SERVER] and flag [--all] cannot be both specified.")
 				}
 				server = args[0]
 			}
+			logging.S.Debugf("flags set:")
```

Comment:
We can remove these debug statements, since the flags being set is part of the input and printing this may not help the user much

Reply:
done